libs/menu-matriarch/feature-ingredients/src/lib/ingredients-board/ingredients-board-column/ingredients-board-column.component.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/member-ordering */
+import { CommonModule } from '@angular/common';
+import {
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+  ChangeDetectionStrategy,
+} from '@angular/core';
+import {
+  CdkDragDrop,
+  DragDropModule,
+  moveItemInArray,
+  transferArrayItem,
+} from '@angular/cdk/drag-drop';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+
+import { IngredientCardComponent } from '../../ingredient-card/ingredient-card.component';
+import { IngredientsBoardFormComponent } from './ingredients-board-form.component';
+import { Ingredient } from '@atocha/menu-matriarch/util';
+import { ingredientTrackByFn } from '@atocha/menu-matriarch/ui-domain';
+
+export interface ColumnMove {
+  columnName: string;
+  currentIndex: number;
+  previousIndex: number;
+}
+
+export interface IngredientAdd {
+  ingredientName: string;
+  columnName: string;
```

Comment:
Pass in `columnId` -- even though it's not needed for now, it’s more future-proof in case there's a future possibility of the display names being the same. Continue using the column name as the ID for now since it's unique and not configurable

Reply:
On second thought: YAGNI. This can be implemented at a later time when needed